LogicalTypes.md

```diff
@@ -148,3 +148,203 @@ primitive type. The `binary` data is interpreted as an encoded BSON document as
 defined by the [BSON specification][bson-spec].
 
 [bson-spec]: http://bsonspec.org/spec.html
+
+## Nested Types
+
+This section specifies how `LIST` and `MAP` can be used to encode nested types
+by adding group levels around repeated fields that are not present in the data.
+
+This does not affect repeated fields that are not annotated: A repeated field
+that is neither contained by a `LIST`- or `MAP`-annotated group nor annotated
+by `LIST` or `MAP` should be interpreted as a required list of required
+elements where the element type is the type of the field.
+
+### Lists
+
+`LIST` is used to annotate types that should be interpreted as lists.
+
+`LIST` must always annotate a 3-level structure:
+
+```
+<list-repetition> group <name> (LIST) {
+  repeated group list {
+    <element-repetition> <element-type> element;
+  }
+}
+```
+
+* The outer-most level must be a group annotated with `LIST` that contains a
+  single field named `list`. The repetition of this level must be either
+  `optional` or `required` and determines whether the list is nullable.
+* The middle level, named `list`, must be a repeated group with a single
+  field named `element`.
+* The `element` field encodes the list's element type and repetition.
```

Comment:
this one is either required or optional (if the list can contain null elements) but can not be repeated